src/dex-ui/pages/dao/GovernanceDAODashboard/GovernanceDAODashboardOverview.tsx

```diff
@@ -0,0 +1,168 @@
+import { Flex, Grid, GridItem, Text } from "@chakra-ui/react";
+import { Color, GlobeIcon, InstagramIcon, MetricLabel, TwitterIcon } from "@dex-ui-components";
+import { useOutletContext } from "react-router-dom";
+import { GovernanceDAODetailsContext } from "./types";
+
+export function GovernanceDAODashboardOverview() {
+  const { dao, totalAssetValue, tokenCount } = useOutletContext<GovernanceDAODetailsContext>();
+  return (
+    <Flex gap="8" direction="column">
+      <Flex gap="4" direction="column">
+        <Text textStyle="h4 medium">Overview</Text>
+        <Grid templateColumns="repeat(2, 1fr)" gap={2}>
+          <GridItem>
+            <Flex
+              direction="column"
+              width="100%"
+              height="100%"
+              bg={Color.White_02}
+              justifyContent="space-between"
+              border={`1px solid ${Color.Neutral._200}`}
+              borderRadius="4px"
+              padding="1.5rem"
+              gap="8"
+            >
+              <Text textStyle="p medium semibold">Assets</Text>
+              <Flex direction="row">
+                <Flex flex={1}>
+                  <MetricLabel
+                    label="TOTAL ASSETS"
+                    labelTextColor={Color.Neutral._500}
+                    labelTextStyle="p xsmall medium"
+                    labelOpacity="1.0"
+                    value={totalAssetValue ?? 0}
+                    valueStyle="p large semibold"
+                    valueTextColor={Color.Grey_Blue._800}
+                    valueUnitSymbol="USD"
+                    valueUnitSymbolColor={Color.Grey_Blue._800}
+                  />
+                </Flex>
+                <Flex flex={1}>
+                  <MetricLabel
+                    label="TOKENS"
+                    labelTextColor={Color.Neutral._500}
+                    labelTextStyle="p xsmall medium"
+                    labelOpacity="1.0"
+                    value={tokenCount ?? 0}
+                    valueStyle="p large semibold"
+                    valueTextColor={Color.Grey_Blue._800}
+                  />
+                </Flex>
+              </Flex>
+            </Flex>
+          </GridItem>
+          <GridItem>
+            <Flex
+              direction="column"
```

Comment:
You can use `layerStyle` variants from our custom ChakraUI theme: `<Flex layerStyle="content-box" />`. You can view/modify the existing styles under `dex-ui-components/themes/layerStyles`

Reply:
Added a new variant dao__dashboard__card in layer styles which extends base style and used it here